sympy/geometry/parabola.py

```diff
@@ -0,0 +1,352 @@
+"""Parabolic geometrical entity.
+
+Contains
+* Parabola
+
+"""
+
+from __future__ import division, print_function
+
+from sympy.core import S
+from sympy.core.numbers import oo
+
+from sympy.geometry.entity import GeometryEntity, GeometrySet
+from sympy.geometry.point import Point
+from sympy.geometry.line import Line
+from sympy.geometry.util import _symbol
+
+
+class Parabola(GeometrySet):
+    """A parabolic GeometryEntity.
+
+    A parabola is declared with a point, that is called 'focus', and
```

Comment:
Other instantiations could be implemented (now or later): vertex and point or three points and direction or no arguments to create the canonical vertical parabola (`y=x**2`) -- like the canonical triangle created as `Triangle(sss=(1,1,1))` -- and then create a specific case with rotation or scaling.